presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/Util.java

```diff
@@ -71,6 +71,7 @@ public static PlanNode transpose(PlanNode parent, PlanNode child)
                 parent.replaceChildren(
                         child.getSources())));
     }
+
```

Comment:
Should be fixed in a39d73de31e3f2972820b68ba2232d5050f093bb (and original PR) instead.